kube-core/src/request.rs

```diff
@@ -247,10 +252,13 @@ impl Request {
 /// additional parameters that retrieve only necessary metadata from an object.
 impl Request {
     /// Get a single metadata instance for a named resource
-    pub fn get_metadata(&self, name: &str) -> Result<http::Request<Vec<u8>>, Error> {
+    pub fn get_metadata(&self, name: &str, gp: &GetParams) -> Result<http::Request<Vec<u8>>, Error> {
         let target = format!("{}/{}", self.url_path, name);
         let mut qp = form_urlencoded::Serializer::new(target);
         let urlstr = qp.finish();
+        if let Some(rv) = &gp.resource_version {
+            qp.append_pair("resourceVersion", rv);
```

Comment:
Ah, one minor point where the codecov actually is handy. I'd expect at least one unit test setting an explicit rv in request.rs using one of the getparam ctors, and verifying that it's part of the qs.

Reply:
There's also some minor doc test failures.

Reply:
Ah, unit tests to the rescue. We were missing a `?` in the target url.